django_celery_extensions/task.py

```diff
@@ -34,20 +34,38 @@
 cache = caches[settings.CACHE_NAME]
 
 
-def default_unique_key_generator(task, task_args, task_kwargs):
+def default_unique_key_generator(task, prefix, task_args, task_kwargs):
     task_args = task_args or ()
     task_kwargs = task_kwargs or {}
 
     _, _, data = serialization.dumps(
         (list(task_args), task_kwargs), task._get_app().conf.task_serializer,
     )
-    return str(uuid.uuid5(uuid.NAMESPACE_DNS, ':'.join((settings.KEY_PREFIX, task.name, data))))
+    return str(uuid.uuid5(uuid.NAMESPACE_DNS, ':'.join((prefix, task.name, data))))
 
 
 class NotTriggeredCeleryError(CeleryError):
     pass
 
 
+class IgnoredResult:
```

Comment:
Yes but unfortunatelly in the celery code there is no abstract result class. There is only AsyncResult but this is exactly the class which I don't want to return. This only simulates similar behavior. If I will use AsyncResult I will have to override all its methods with some exeption body. I can use BaseResult from celery but this class has no meaning so this will be scam on you

I don't like it too, but there is no other option

Reply:
OK, i didn't notice that the other class is in Celery.